.env.example

```diff
@@ -8,3 +8,6 @@
 # ANVIL_API_KEY=IzqBEyxFCTiO7jwcfEihGww5AFXL5CAh
 
 ANVIL_API_KEY=
+# For local development uncomment below
+# API_BASE_URL=http://localhost:8080
+# BASE_URL=http://localhost:3001
```

Comment:
Remove these? I think it will cause confusion. For our local dev the API_BASE_URL needs to be :3000 and BASE_URL probably needs to be :8080, but should be the default in the config index for easy setup on people's local kits.

Reply:
I'll clean it up. It should default to the dev situation for sure.